jest.setup.js

```diff
@@ -40,4 +41,5 @@ beforeEach(() => {
 
 afterEach(() => {
   ReactDOM.unmountComponentAtNode(mount);
+  clearInstances();
```

Comment:
Because instances is now not tied to the HelmetProvider instance, but a singleton on the page, it needs to be reset at the end of each test.